frontend/src/components/containers/ContainerFilters.tsx

```diff
@@ -0,0 +1,39 @@
+import React from "react";
+import {filter as filterObject} from "rambda";
+import {Collapse} from "antd";
+
+import {setFilter} from "../../modules/containers/actions";
+import FilterGroup from "../filters/filterGroup/FilterGroup";
+import {CONTAINER_FILTERS} from "../filters/descriptions";
+import { useAppDispatch, useAppSelector } from "../../hooks";
+import { FilterDescription, FilterSetting, FilterValue } from "../../models/paged_items";
+
+type ContainerFiltersProps = Pick<React.DetailedHTMLProps<React.HTMLAttributes<HTMLDivElement>, HTMLDivElement>, 'style'>
+
+const ContainerFilters = (rest: ContainerFiltersProps) => {
+  const filters = useAppSelector((state) => state.containers.filters as FilterSetting[])
+  const dispatch = useAppDispatch()
+
+  const onChangeFilter = (filter: FilterDescription, value: FilterValue) => {
+    dispatch(setFilter(filter.key, value))
+  }
+
+  return (
+    <div className='SamplesFilters' {...rest}>
+      <Collapse defaultActiveKey={[]} ghost>
```

Comment:
It would be better if we use `collapsible="header"` so that the Collapse only expands if the user clicks on the 'Show advanced filters' text. I think that should also be fixed in the Samples table. I don't know how many times I've clicked in the white area at the top of the view and accidentally expanded the custom filters section...